tests/frontend/verifier/test_verifier_argoverse.py

```diff
@@ -89,10 +89,18 @@ def check_verifier_output_error(verifier: VerifierBase, euler_angle_err_tol: flo
 
     euler_angles = Rotation.from_matrix(i1Ri2).as_euler("zyx", degrees=True)
     gt_euler_angles = np.array([-0.37, 32.47, -0.42])
-    assert np.allclose(gt_euler_angles, euler_angles, atol=euler_angle_err_tol)
+    gt_euler_angles_str = str(np.round(gt_euler_angles, 1))
+    euler_angles_str = str(np.round(euler_angles, 1))
+    assert np.allclose(
+        gt_euler_angles, euler_angles, atol=euler_angle_err_tol
+    ), f"GT {gt_euler_angles_str} vs. Est. {euler_angles_str} w/ tol {euler_angle_err_tol:.1f}"
 
     gt_i1ti2 = np.array([0.21, -0.0024, 0.976])
-    assert np.allclose(gt_i1ti2, i1ti2, atol=translation_err_tol)
+    gt_i1ti2_str = str(np.round(gt_i1ti2, 1))
+    i1ti2_str = str(np.round(i1ti2, 1))
```

Comment:
there should be a way to do this with python string formatting itself I think, like the 0.1f format.